Week1/homework/app.js

```diff
@@ -1,11 +1,171 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // 1.1
+  // const bookTitles = [
+  //   'harry_potter_chamber_secrets',
+  //   'sufisim',
+  //   'the_statue_of_our_souls',
+  //   'the_words',
+  //   'safahat',
+  //   'towards_the_lost_paradise',
+  //   'sapiens',
+  //   'facade',
+  //   'love_and_tolerance',
+  //   'from_seed_to_cedar',
+  // ];
+
+  const rootDiv = document.getElementById('root');
+
+  // 1.3
+  // function getBooks(arr) {
+  //   const bookList = document.createElement('ul');
+  //   rootDiv.appendChild(bookList);
+
+  //   for (const elem of arr) {
+  //     const bookTitle = document.createElement('li');
+  //     bookTitle.innerHTML = elem;
+  //     bookList.appendChild(bookTitle);
+  //   }
+  // }
+  // getBooks(bookTitles); it is for question 1.3
+
+  const completeBookList = [
```

Comment:
You create an array instead of an object as was asked in the instructions. Please look at the example I provided during the class.
You can find it here in the video: https://youtu.be/xKekzhG1kKM?t=108

Reply:
I did it sir.